pkg/controller/kubedirectorcluster/cluster.go

```diff
@@ -199,11 +188,11 @@ func syncCluster(
 // reject old/stale versions of the CR.
 func handleStatusGen(
```

Comment:
Some changes to this function proposed in my long comment.

Reply:
Done.